src/v/kafka/server/group_manager.cc

```diff
@@ -625,7 +639,18 @@ ss::future<> group_manager::handle_partition_leader_change(
     return p->catchup_lock->hold_write_lock().then(
       [this, term, timeout, p](ss::basic_rwlock<>::holder unit) {
           return inject_noop(p->partition, timeout)
-            .then([this, term, timeout, p] {
+            .then([this, term, timeout, p](std::error_code error) {
+                if (error) {
+                    vlog(
+                      klog.warn,
+                      "unable to inject linearizable barrier for partition: {} "
+                      "consumer groups state recovery: {}",
```

Comment:
"... partition <ntp> consumer groups state recovery: <error>" doesn't sound right to me - should be something like "... consumer groups state partition <ntp>, error: <error>", no?